apps/core/lib/core/pubsub/protocols/hoggable.ex

```diff
@@ -19,3 +19,7 @@ defimpl Core.PubSub.Hoggable, for: [Core.PubSub.InstallationCreated, Core.PubSub
   defp event(PubSub.InstallationCreated), do: "installation.created"
   defp event(PubSub.InstallationDeleted), do: "installation.deleted"
 end
+
+defimpl Core.PubSub.Hoggable, for: Core.PubSub.UserCreated do
+  def hog(%{item: user}), do: {"user.created", user.id, %{email: user.email}}
```

Comment:
We can’t set the user email here since we’d need to check consent for that (which is done with a cookie on the frontend). But this shouldn’t be a problem since if the user accepts cookies their email will get associated with this ID in PostHog later on.

Reply:
Shouldn't that become null if they sign up though (I'm not sure you can even sign up w/o cookies actually)

Reply:
You can signup without cookie tracking. It won’t become null, since we use the Plural UUID to track users. So if they never consent to cookies we would have marketing site visits of a user that aren’t associated with this signup event. However, as soon as they consent all the anonymous PostHog IDs will get associated with the Plural UUID and it will set the user’s email on their person in PostHog.